internal/model/board.go

```diff
@@ -190,6 +189,14 @@ func (b *Board) MoveCurrentPlayerTo(line, row int) error {
 	currentPlayer := b.CurrentPlayer()
 	currentPlayer.Line = line
 	currentPlayer.Row = row
+
+	currentTile := b.Tiles[line][row]
+	if currentTile.Tile.Treasure == currentPlayer.Hand[0] {
+		currentPlayer.Hand = currentPlayer.Hand[1:]
```

Comment:
Can you explaim me what this assignment does ?

Reply:
This is the equivalent of `shift` in javascript, since Go does not provide a `Shift` method